src/endpoint/routes.ts

```diff
@@ -137,11 +122,11 @@ export default (server, createFactory) => {
         await nodes[id].start()
 
         return {
-          apiAddr: nodes[id].apiAddr ? nodes[id].apiAddr.toString() : '',
-          gatewayAddr: nodes[id].gatewayAddr ? nodes[id].gatewayAddr.toString() : '',
-          grpcAddr: nodes[id].grpcAddr ? nodes[id].grpcAddr.toString() : ''
+          apiAddr: nodes[id].apiAddr?.toString(),
+          gatewayAddr: nodes[id].gatewayAddr?.toString(),
+          grpcAddr: nodes[id].grpcAddr?.toString()
```

Comment:
we're changing the results from being `''` (empty string) when undefined, to being `undefined`. Is this okay?

Reply:
Yes, I think it's a bit better like as these fields will be stripped out on JSON encode now as they're undefined instead of being an empty string which is not a valid string representation of a multiaddr.